### DISCUSSION TITLE
I was approved by github education pack but i dont get the access to it help

### DISCUSSION THREAD

**Discussion by Guru12e** [community/community#107530] [GitHub Education]
### Hi everyone,

I was approved by github education pack but i dont get the access to it help i was approved by yesterday but i dont get access to it help me.

**Comment by mw-xyz** [comment]
They should be available [here](https://education.github.com/pack/offers) but if not, make sure you're logged in with the same account, and that you're using the same email(s) you used for verification. If you go there and it says you haven't signed up, definitely try the above. Let me know if this helps
